pkgs/misc/emulators/yuzu/base.nix

```
@@ -0,0 +1,72 @@
+{ pname, version, src, branchName
+, stdenv, lib, fetchFromGitHub, wrapQtAppsHook
+, cmake, pkg-config
+, libpulseaudio, libjack2, alsaLib, sndio, ecasound
+, vulkan-loader, vulkan-headers
+, qtbase, qtwebengine, qttools
+, nlohmann_json, rapidjson
+, zlib, zstd, libzip, lz4
+, glslang
+, boost173
+, catch2
+, fmt
+, SDL2
+, udev
+, libusb1
+, ffmpeg
+}:
+
+stdenv.mkDerivation rec {
+  inherit pname version src;
+
+  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
+  buildInputs = [
+    libpulseaudio libjack2 alsaLib sndio ecasound
+    vulkan-loader vulkan-headers
+    qtbase qtwebengine qttools
+    nlohmann_json rapidjson
+    zlib zstd libzip lz4
+    glslang
+    boost173
+    catch2
+    fmt
+    SDL2
+    udev
+    libusb1
+    ffmpeg
+  ];
+
+  cmakeFlags = [
+    "-DENABLE_QT_TRANSLATION=ON"
+    "-DYUZU_USE_QT_WEB_ENGINE=ON"
+    "-DUSE_DISCORD_PRESENCE=ON"
+  ];
+
+  # Trick the configure system. This prevents a check for submodule directories.
+  preConfigure = ''
+    rm -f .gitmodules
```

```suggestion
    rm .gitmodules
```

We want to notice when that file is no longer there.

---

As I've stated in your original review, not all branches of yuzu have this file present. For example, the "yuzu-ea" one we use in this derivation does not. Your suggested change causes a build failure on this branch.